webcompat/templates/contributors/diagnose-bug.html

```diff
@@ -1,11 +1,14 @@
 <ul class="contributors__list">
-  <li class="contributors__list__item"><span>Find a bug with a "needsdiagnosis" label or that is otherwise new</span></li>
-  <li class="contributors__list__item"><span>Check to see if bug has been reproduced already</span></li>
+  <li class="contributors__list__item"><span>Find a bug that "<a href="https://webcompat.com/issues?page=1&per_page=50&state=open&stage=needsdiagnosis&sort=created&direction=desc">Needs Diagnosis or is otherwise new</a>"</span>
```

Comment:
Hmm, I think this is sort of out of date. Before we had this notion of "new" bugs. But now we call them "needs triage". Can we restrict the hyperlink to just `Needs Diagnosis` and add a link for "Needs Triage" (instead of "otherwise new")?